src/test/php/PDepend/Bugs/ParserBug713Test.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace PDepend\Bugs;
+
+use PDepend\Source\Language\PHP\PHPBuilder;
+use PDepend\Source\Language\PHP\PHPParserGeneric;
+use PDepend\Source\Language\PHP\PHPTokenizerInternal;
+use PDepend\Util\Cache\Driver\MemoryCacheDriver;
+
+/**
+ * Test case for bug #713.
+ *
+ * @ticket 713
+ * @covers \stdClass
```

Comment:
That doesn't make much sense. Let's better state the actual coverage:

```suggestion
 * @covers \PDepend\Source\Language\PHP\AbstractPHPParser::parseIssetExpression
```